README.md

```diff
@@ -172,6 +172,25 @@ dependencies.
 
 `acbuild run` also requires root.
 
+## Signing ACIs
+
+When finishing a build with `acbuild end`, if the `--sign` flag is provided
+acbuild will sign the resulting ACI by invoking the `gpg` command on the
+system. By default the `--armor` and `--yes` flags are passed to `gpg`, but if
+this is not sufficient any arguments given to `acbuild end` after the ACI to
+write are passed along to `gpg`.
+
+The `--user` flag may also be specified for the user to run the `gpg` command
+as. This is provided to allow running `end` as root, which may be necessary to
+clean up files created by the root user during a `run` command, while still
+using the user's keyring.
+
+An example of this functionality is:
+
+```
+acbuild end --sign mynewapp.aci -- --no-default-keyring --keyring ./rkt.gpg
```

Comment:
I'm almost wondering about splitting this out into two steps - `acbuild end` and `acbuild cleanup`. It just feels awkward to run an end command as root.